ansible/roles/debops.grafana/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Restart grafana
+  service:
+    name: 'grafana-server'
+    state: 'restarted'
+    enabled: yes
```

Comment:
Setting `enabled: True` here might have issues on installations, where `grafana.service` was intentionally disabled by system administrator. The service will probably be enabled by default on installation, so you don't need this parameter here.

Reply:
deleted enabled entry